person_test.go

```diff
@@ -53,11 +53,13 @@ func TestLastName(t *testing.T) {
 func TestName(t *testing.T) {
 	p := New().Person()
 	name := p.Name()
+	gender := p.Gender()
```

Comment:
just one little change here if possible, can you create a `func TestGender(t *testing.T)` for this? so we can keep it isolated from the testing name.